connectivity/drivers/ble/FEATURE_BLE/TARGET_NORDIC/CMakeLists.txt

```
@@ -1,4 +1,8 @@
 # Copyright (c) 2020 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
+add_library(mbed-os-ble-nordic-cordio OBJECT)
```

Why not
```suggestion
add_library(mbed-os-ble-nordic OBJECT)
```
since you are not in the cordio dir here?